estate/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "estate",
+
+    'description': """
+        Module to deal with estate sales"
+    """,
+
+    'author': "Gaso",
```

Comment:
Just a random comment to let you know that you can't do that in the official repo.
```suggestion
    'author': 'Odoo S.A.',
```